src/validate.js

```diff
@@ -152,13 +156,19 @@ export default function validateFormData(
   formData,
   schema,
   customValidate,
-  transformErrors
+  transformErrors,
+  additionalMetaSchemas
 ) {
+  // add more schemas to validate against
+  if (!addedMetaSchemas && additionalMetaSchemas) {
+    ajv.addMetaSchema(additionalMetaSchemas);
```

Comment:
Technically, this still allows `additionalMetaSchemas` to not be an array.

Reply:
Yeah, good catch, fix'd 👍